syntax = "proto3";

message TaskRequest {
  int32 task_count = 1;
}

message TaskResponse {
  string task = 1;
  bool error = 2;
}

service Task {
  rpc GetTask(TaskRequest) returns (TaskResponse);
}
